app/apollo/resolvers/resource.updateClusterResources.js

```diff
@@ -0,0 +1,296 @@
+/**
+ * Copyright 2020 IBM Corp. All Rights Reserved.
```

Comment:
New files should have a copyright of 2021 rather than 2020.

_Existing_ files that are updated also need copyright statements updated appropriately.